hc/front/tests/test_add_channel.py

```diff
@@ -37,5 +37,24 @@ def test_instructions_work(self):
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
 
+    def test_bad_kinds_not_working(self):
+        self.client.login(username="alice@example.org", password="password")
+        bad_kind = "twitter"
+        url = "/integrations/add_%s/" % bad_kind
+        r = self.client.get(url)
```

Comment:
Use a more descriptive variable name, in place of `r`. See [here](http://pep8.org/#names-to-avoid)

Reply:
And any other line you have similar variables. Like this [one](https://github.com/andela/healthchecks-f-society/blob/0a00ed2b0eb44ad02e2d4bea785ec04bce80ca79/hc/front/tests/test_add_pushbover.py#L56)